src/ripple/ledger/ViewAPI.h

```diff
@@ -208,8 +214,9 @@ trustCreate (View& view,
                                         // Issuer should be noAccount()
     STAmount const& saLimit,            // --> limit for account being set.
                                         // Issuer should be the account being set.
-    const std::uint32_t uSrcQualityIn = 0,
-    const std::uint32_t uSrcQualityOut = 0);
+    std::uint32_t uSrcQualityIn,
+    std::uint32_t uSrcQualityOut,
+    CreditHook hook);
```

Comment:
Consider a setting default parameter for `hook`. Many callers will want to call this as `trustCreate(...,{});`